packages/react/src/Suspense.tsx

```diff
@@ -21,10 +21,10 @@ if (process.env.NODE_ENV !== 'production') {
   DefaultSuspense.displayName = 'Suspense'
 }
 const CSROnlySuspense = (props: SuspenseProps) => {
-  const isMounted = useIsMounted()
+  const isClient = useIsCSROnly()
```

Comment:
```suggestion
  const isClient = useIsClient()
```

Can we change the name of this hook to useIsClient? If this hook is likely to be reused elsewhere in the future, I don't think it will only be used in CSROnly. In that case, I think a better name would be useIsClient rather than CSROnly. Looking at the code, the return value is also the variable name isClient, so I think useIsClient would be a better name. Because the name CSROnly also means Client Side Render Only, which can contain isClient.

Reply:
Im down

Then our Suspense 
Should it be `Suspense.Client`, not `Suspense.CSROnly`?

Reply:
@okinawaa I think`<Suspense.CSROnly/>` using `useIsClient` is best case.
then we will not go through `BREAKING CHANGE` and also naming this component as `<Suspense.Client/>` is not more clear to express `client side render only` than `<Suspense.CSROnly/>`.

Reply:
@manudeli 
I agree with you. 

CSROnly naming will keep, and useIsCSROnly naming is will changed to useIsClient. I will do that

Reply:
Thanks for accepting my opinion